### DISCUSSION TITLE
Add npm-check ?

### DISCUSSION THREAD

**Discussion by lakgani** [sindresorhus/awesome-nodejs#1102] [Ideas]
I find [npm-check](https://github.com/dylang/npm-check) as a very valuable tool to keep the node projects up to date. With a single command we can see all the packages in the project that are outdated and also provides the ability to do upgrades in interactive mode. Do you think it would be useful to add them to the list?

Adding the screenshot from `npm-check` repo below to give an idea of the interactive UI it provides

![](https://cloud.githubusercontent.com/assets/51505/9569917/96947fea-4f48-11e5-9783-2d78077256f2.png)

**Comment by silviuburceadev** [comment]
`npm outdated` gives you the same thing, it's just not interractive.
